packages/components/src/private-apis.ts

```diff
@@ -28,11 +22,11 @@ import { lock } from './lock-unlock';
 
 export const privateApis = {};
 lock( privateApis, {
-	CompositeV2,
-	CompositeGroupV2,
-	CompositeItemV2,
-	CompositeRowV2,
-	useCompositeStoreV2,
+	CompositeV2: Composite.Root,
```

Comment:
We're planning to start using the stable imports and get rid of those in a separate PR, right?

Reply:
Yup, there are 2 follow-up PRs planned:
- one where we swap all usages to the public APIs and delete the private APIs
- one where we deprecated the V1 __unstable APIs